lib/authoritex/mock.ex

```diff
@@ -46,9 +52,13 @@ defmodule Authoritex.Mock do
   def can_resolve?(_id), do: true
 
   @impl Authoritex
+  def fetch("missing_id_authority:" <> _id) do
+    {:error, 404}
+  end
```

Comment:
If we really want this to be consistent with how `Authoritex` works in non-mocked mode, we need to rewrite `Mock.set_data/1` to include a list of authority stems that will be checked on `fetch` to determine whether the authority is unknown or just the ID. Approving as is, but opening an issue for future work.